lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -1114,6 +1114,13 @@ namespace Js
         whackString = CreateStringFromCppLiteral(_u("/"));
         commaDisplayString = CreateStringFromCppLiteral(_u(","));
         commaSpaceDisplayString = CreateStringFromCppLiteral(_u(", "));
+        openBracketString = CreateStringFromCppLiteral(_u("{"));
```

Comment:
>openBracketString [](start = 8, length = 17)

Similar things are defined in JavascriptFunction.cpp, since your are cleaning things good to remove those as well.

[https://github.com/Microsoft/ChakraCore/blob/master/lib/Runtime/Library/JavascriptFunction.cpp#L148](https://github.com/Microsoft/ChakraCore/blob/master/lib/Runtime/Library/JavascriptFunction.cpp#L148)